src/components/RocketsDependencies/ReserveButtonComponent.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import PropTypes from 'prop-types';
+import { Button }  from "react-bootstrap";
+
+
+const ReserveButton = ({ onClickEvent, reserved, rocketID }) => {
+  let text; let variant;
+  if (reserved){
+    text = "Cancel Reservation"
+    variant = "secondary"
+  } else {
+    text = "Reserve"
+    variant = "primary"
+  }
```

Comment:
- [ ] [OPTIONAL] I would recommend using the ternary operator. This would help you reduce these 8 lines to 2 lines and also would make the code look cleaner.